.github/workflows/ci.yaml

```diff
@@ -40,7 +40,38 @@ jobs:
       - name: Build
         timeout-minutes: 40
         run: make build
-  
+
+  e2e-test:
+    name: e2e test
+    runs-on: ubuntu-latest
+
```

Comment:
this does not seem to be doing anything usimg the sdf-http crate

Reply:
this is not completed yet